contracts/seed/Seed.sol

```diff
@@ -55,6 +55,8 @@ contract Seed {
     bool    public initialized;            // is this contract initialized [not necessary that it is funded]
     bool    public minimumReached;         // if the softCap[minimum limit of funding token] is reached
     bool    public maximumReached;         // if the hardCap[maximum limit of funding token] is reached
+    uint256 public vestingStartTime;       // timestamp when vesting start, by default = endTime,
```

Comment:
"// timestamp for when vesting starts, by default == endTime, otherwise when maximumReached is reached"